packages/wasm/package.json

```diff
@@ -3,12 +3,12 @@
   "version": "1.0.0",
   "description": "",
   "type": "module",
-  "main": "dist/nodejs/wasm.cjs",
```

Comment:
Are we losing anything dropping this? If I recall correctly, just support for projects using older versions of node?

Reply:
yeah and next ignores export maps and uses main for SSR ...